File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/3040.rb
n = gets.to_i

for i in 1..n do
    ns = gets.chomp().split(' ')
    a = ns[0].to_i   
    b = ns[1].to_i
    c = ns[2].to_i
    if(a>=200 and a<=300 and b>=50 and c>=150) then
        puts("Sim")
    else
        puts("Nao")
    end
end

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/3040.lua
n = io.read("*number")

for i=1,n do
    a = io.read("*number")
    b = io.read("*number")
    c = io.read("*number")
    if(a>=200 and a<=300 and b>=50 and c>=150) then
        print("Sim")
    else
        print("Nao")
    end
end